recruiter/templates/recruiter/registerRecruiter.html

```diff
@@ -9,7 +9,11 @@ <h1>Register As Recruiter</h1>
     <button class="btn btn-primary" type="submit">Register</button>
   </form>
   <p class="text-center">
-    If you already have an account, <a href="/login">login</a> instead.
+    If your company isn't in the list add a new company
+    <a href="/companyRegister">here</a>.
```

Comment:
@YarinBou we can change it to 
`If your company isn't in the list,
    <a href="/companyRegister">add a new company</a>.

Reply:
we can, but why should we? that's not really change anything

Reply:
ok, lets leave it that way then